tests/integration/mock/CodeActAgent/test_browse_internet/prompt_001.log

```diff
@@ -401,4 +401,4 @@ NOW, LET'S START!
 
 Browse localhost:8000, and tell me the ultimate answer to life. Do not ask me for confirmation at any point.
 
-ENVIRONMENT REMINDER: You have 14 turns left to complete the task. When finished reply with <finish></finish>
+ENVIRONMENT REMINDER: You have 49 turns left to complete the task. When finished reply with <finish></finish>
```

Comment:
This seems problematic. It looks like it's using max_iterations = 50, presumably from your `config.toml`. 

In `regenerate.sh` we defined it as `MAX_ITERATIONS=15`

Reply:
Sorry, this is fixed!